### PR TITLE
My Bench: Conditional Rendering 

### PR DESCRIPTION
**Pull request by ae-andre**
#Primary changes 

- Added conditional rendering for the My Bench page
     - My Bench will display 'conversationCard' if a conversation already exists
     - Cards.jsx includes logic to dynamically render 'conversationCards' if we can support a user having multiple conversations at once (incl. conversation history)
     - In the same vein, a card is presented to the user to start a new conversation on the My Bench place as a static element 

image

- I set up this logic to conditionally render either the Cards OR the ConversationsForm, but it might be worth rethinking the flow since we can use the Start New Convo card instead:

image

- For above, I'd recommend a new page for the start new convo form

-------------------
##Minor details / Notes

- Added logic to limit characters shown on cards
- Once the matching logic works, the Cards.jsx can be adjusted to display the buddy's name
- BUG: From /my-bench, the logout button does not redirect away from my-bench, so it causes an error saying the validation token is invalid:

image

### CODE REVIEW

File: client/src/components/Cards.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            <dd className="mb-8 p-1 flex-col items-baseline sm:pb-3">
+              <p className="flex justify-center text-med font-medium bg-orange-100 rounded-md text-gray-800">
+                {/* -------------limit the characters to 33 max------------- */}
+                {item.preview.slice(0, 37)}
+                {item.preview.length > 37 ? "..." : ""}
+              </p>
+              <div className="absolute inset-x-0 bottom-0 px-4 py-4 sm:px-6">
+                <div className="text-sm">
+                  <p className="mb-2 text-sm font-medium text-gray-500">
+                    Your sharing this bench with {item.buddy}
```
Comment (tchan128) [inline client/src/components/Cards.jsx:89]: @ae-andre 

> >Once the matching logic works, the Cards.jsx can be adjusted to display the buddy's name

Is this line not accurately pulling the buddy username?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "tech-friends",
+  "name": "soul-bench",
```
Comment (tchan128) [inline package.json:2]: nice catch, thanks for changing that!

**Comment by tchan128** [comment]
> I set up this logic to conditionally render either the Cards OR the ConversationsForm, but it might be worth rethinking the flow since we can use the Start New Convo card instead:

@ae-andre I am guessing this means that the current flow is:

-> Yes conversation will render the cards components including the + a new conversation & the current convo
-> No conversation will render the conversation form

**Review by tchan128 — APPROVED** [review]

**Review by kwubbenhorst — APPROVED** [review]
Very nice UI. Good work.

**Pull request merged**